templates/toil-cluster-role-and-policy.yaml

```diff
@@ -41,6 +41,7 @@ Resources:
               - ec2:CreateSecurityGroup
               - ec2:CreateTags
               - ec2:DeleteSecurityGroup
+              - ec2:DeleteTags
```

Comment:
hmm, i don't see this one in `TagRootVolumeRole` https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/essentials.yaml#L323

do we need it?

Reply:
You're right, we don't, I'll remove it.